ssh-to-spacestation.sh

```diff
@@ -0,0 +1 @@
+trickle -s -d 5 -u 5 -L 400 ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $STATION_SSH_KEY $STATION_USERNAME@$STATION_CONTAINER_NAME
```

Comment:
Could use a shebang?

```suggestion
#!/bin/bash
trickle -s -d 5 -u 5 -L 400 ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $STATION_SSH_KEY $STATION_USERNAME@$STATION_CONTAINER_NAME
```